docs/postpaid/response-code.md

```diff
@@ -2,52 +2,52 @@
+33 | TRANSACTION CAN'T BE PROCESS, PLEASE TRY AGAIN LATER | Failed | There is an error on your body field. Please check again your product code for typos, or retry with other product code.
+34 | BILL HAS BEEN PAID | Failed | Your bill for your current transaction has been paid. Please try again for another transaction.
+35 | TRANSACTION REJECTED DUE TO ANOTHER UNPAID ARREAR | Failed | Your transaction id failed. Please pay for your other arrear first, then try again your transaction.
+36 | EXCEEDING DUE DATE, PLEASE PAY IN THE COUNTER / PDAM | Failed | Your current transaction is exceeding the due date. Please pay the transaction at the counter.
+37 | PAYMENT FAILED | Failed | Your payment failed, please try again later.
+38 | PAYMENT FAILED, PLEASE DO ANOTHER REQUEST | Failed | Your payment failed, please try again with a new request.
+39 | PENDING / TRANSACTION IN PROCESS | Pending | Your transaction is still being processed. Please wait for further status updates.
+40 | TRANSACTION REJECTED DUE TO ALL OR ONE OF THE ARREAR/INVOICE HAS BEEN PAID | Failed | Your transaction has already been paid. You can try again with another transaction.
+41 | CAN'T BE PAID IN COUNTER, PLEASE PAY TO THE CORRESPONDING COMPANY | Failed | Your current transaction cannot be paid in the counter. You can pay your transaction to the corresponding company.
+42 | PAYMENT REQUEST HAVEN'T BEEN RECEIEVED | Failed | Your current transaction is still in the inquiry process. Please continue your payment process first.
+91 | DATABASE CONNECTION ERROR | Failed | There is an error on the database connection. Please try again later.
+92 | GENERAL ERROR | Failed | The received response code is undefined yet. Please contact our Customer Service.
+93 | INVALID AMOUNT | Failed | The amount inputted isn’t valid or identical with the amount from the supplier. Please check again your inputted amount.
+94 | SERVICE HAS EXPIRED | Failed | -
+100 | INVALID SIGNATURE | Failed | Your sign (signature) field doesn’t contain the right key for your current request. Please check again your sign value.
+101 | INVALID COMMAND | Failed | The command that you’ve inputted is not a valid command, try checking your commands field for typos or try another command.
+102 | INVALID IP ADDRESS | Failed | Your IP address isn’t allowed to make a transaction. You can add your IP address to your allowed IP address list in https://developer.iak.id/prod-setting.
+103 | TIMEOUT | Failed | Your current request exceeds the timeout limit. You can try to request it again.
+105 | MISC ERROR / BILLER SYSTEM ERROR | Failed | There is an error from the supplier. Please try again later.
+106 | PRODUCT IS TEMPORARILY OUT OF SERVICE | Failed | The product_code that you pick is in non-active status. You can retry your transaction with another product_code that has active status.
```

Comment:
The code is in non-active status. You can retry your transaction with another code that has active status.

Reply:
kasih link pricelist